src/ui/component/userEmail/view.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as React from 'react';
+import Button from 'component/button';
+import { FormField } from 'component/common/form';
+import UserEmailNew from 'component/userEmailNew';
+import UserEmailVerify from 'component/userEmailVerify';
+
+type Props = {
+  cancelButton: React.Node,
+  email: string,
+  resendVerificationEmail: string => void,
+  checkEmailVerified: () => void,
+  user: {
+    has_verified_email: boolean,
```

Comment:
Should user be a first class type and/or should this value simply be selected directly off of the user?

Reply:
Yes to both of these